resources/assets/js/designer/components/modals/modal-create-report-add.vue

```diff
@@ -0,0 +1,330 @@
+            </div>
+          </div>
+        </div>
+    </form>
+
+    <template slot="modal-footer">
+      <b-button @click="onCancel" class="btn-outline-secondary btn-md">
+        CANCEL
+      </b-button>
+      <b-button class="btn-secondary text-light btn-md">
+        SAVE
+      </b-button>
+    </template>
+
+  </b-modal>
+</template>
+
+<script>
+import draggable from 'vuedraggable'
+
+export default {
+  components:{
+    draggable
+  },
+  data() {
+    return {
+      // form models here
+      'description':"Description",
+      'reportTable': "Report Table",
+      'formFields': "Form Fields",
+      'tableName': 'Table Name',
+      'type': 'Type',
+      'typeSelect': "View",
+      'typeSelectOptions':[
+        'View', 'Block'
+      ],
+      'dbConnection':"DB Connection",
+      'dbConnectionSelect': 'View',
+      'dbConnectionSelectOptions':[
+        'View', 'Block'
+      ],
+      list: [
+        {
+          form_field: "F3",
+          form_field_full: "Form Field Name 1",
+          name: "MilaRene",
+          label: "human",
+          type: "yes",
+          size: 16,
+          auto_increment: "nah",
+          index: "cha"
+        },
+        {
+          form_field: "F3",
+          form_field_full: "Form Field Name 1",
+          name: "MilaRene",
+          label: "hello",
+          type: "yes",
+          size: 16,
+          auto_increment: "nah",
+          index: "cha"
+        },
+        {
+          form_field: "F3",
+          form_field_full: "Form Field Name 1",
+          name: "MilaRene",
+          label: "hello",
+          type: "yes",
+          size: 16,
+          auto_increment: "nah",
+          index: "cha"
+        },
+      ],
+
+      list2: [
+        {
+          form_field: "F3",
+          form_field_full: "Form Field Name 1",
+          name: "MilaRene",
+          label: "hello",
+          type: "yes",
+          size: 16,
+          auto_increment: "nah",
+          index: "cha"
+        },
+        {
+          form_field: "F3",
+          form_field_full: "Form Field Name 1",
+          name: "MilaRene",
+          label: "hello",
+          type: "yes",
+          size: 16,
+          auto_increment: "nah",
+          index: "cha"
+        },
+        {
+          form_field: "F3",
+          form_field_full: "Form Field Name 1",
+          name: "MilaRene",
+          label: "hello",
+          type: "yes",
+          size: 16,
+          auto_increment: "nah",
+          index: "cha"
+        }
+      ]
+    }
+  },
+  methods:{
+    onHidden() {
+      this.$emit('hidden')
+    },
+    onCancel() {
+      this.$refs.modal.hide()
+    },
+    add: function() {
+     this.list.push({
+       name: 'Mila'
+     });
+   },
+   replace: function() {
+     this.list = [{
+       name: 'Taylor'
+     }]
+   }
+  },
+  mounted() {
+    // Show our modal as soon as we're created
+    this.$refs.modal.show();
+  }
+};
+</script>
+<style lang="scss" scoped>
+.inline-input{
```

Comment:
Can some of this CSS be folded into the global styles sheet?

Reply:
not that I know of, all of this is specific to this modal